backend_checkouts_test.go

```diff
@@ -0,0 +1,579 @@
+package openldap
```

Comment:
This code is copied from [vault-plugin-secrets-ad](https://github.com/hashicorp/vault-plugin-secrets-ad).